homeassistant/components/wallbox/number.py

```diff
@@ -86,21 +94,29 @@ def __init__(
 
     @property
     def native_max_value(self) -> float:
-        """Return the maximum available current."""
+        """Return the maximum available current. Return a huge monetary value per kWh for price."""
+
+        if self.entity_description.key == CHARGER_ENERGY_PRICE_KEY:
+            return 10.0
+
         return cast(float, self._coordinator.data[CHARGER_MAX_AVAILABLE_POWER_KEY])
 
     @property
     def native_min_value(self) -> float:
-        """Return the minimum available current based on charger type - some chargers can discharge."""
+        """Return the minimum available current based on charger type - some chargers can discharge. Return a huge monetary value per kWh for price."""
+        if self.entity_description.key == CHARGER_ENERGY_PRICE_KEY:
+            return -10.0
+
         return (self.max_value * -1) if self._is_bidirectional else 6
```

Comment:
We might want to include a native_step property, currently hass is doing 1€ steps for this property -> seems drastic ;-) https://developers.home-assistant.io/docs/core/entity/number/